pkg/tbtc/internal/test/testdata/deposit_sweep_scenario_4.json

```diff
@@ -0,0 +1,52 @@
+      "RefundLocktime": "60bcea61",
+      "Vault": "",
+      "ExtraData": "a9b38ea6435c8941d6eda6a46b68e3e2117196995bd154ab55196396b03d9bda"
+    }
+  ],
+  "InputTransactions": [
+    "0100000000010130c1b2951a7019b8d384179b40108d27c02a0be746106dd9aad3532d053a5e650000000000ffffffff01705d1e00000000001600148db50eb52063ea9d98b3eac91489a90f738986f603483045022100c415b9adb4ee0290ba707f5ddd79a9279e2fd54baf973bed5b2482b89a60f082022051e33af64d9958a2de1d52eaa13fe120b5d804f91d770d543afcd7d54bc874c1012103989d253b17a6a0f41838b84ff0d20e8898f9d7b1a98f2564da4cc29dcf8581d95c14a0ee7a142d267c1f36714e4a8f75612f20a7972075083543d400d3653e327576a9148db50eb52063ea9d98b3eac91489a90f738986f68763ac6776a91402551415aa56699043cbc8ab9c8b63efe04a2c0488046f231267b175ac6800000000",
+    "01000000000101352669959363c5bc6248c6fb3c16a1aa5ec630d66672ebb3df00a3c485d5c0a50000000000ffffffff01905f010000000000220020bfaeddba12b0de6feeb649af76376876bc1feb6c2248fbfef9293ba3ac51bb4a02483045022100cf4651094fc0faaf27cd0b7e2f00969960d2b8e65c7211641728d6fa4df203b302206f4da3a78578f5dce5dfc39a57625c472bc72875ed5cfdcaea23a483ff72884e012102ee067a0273f2e3ba88d23140a24fdb290f27bbcd0f94117a9c65be3911c5c04e00000000"
+  ],
+  "Fee": 10000,
+  "Signatures": [
+    {
+      "R": "7a2c2165f22082cd35b78b606ccd9882c0d0b90eca4fc93cb4488875aadb1dd9",
+      "S": "63f77b701286714c392627e9cc69238f8157e56dca4e24e397769c40ed1453d2"
+    },
+    {
+      "R": "f7133a975f9728091fba874f3d6ff23d3c63c084cbb5e3e344c59a7910f96368",
+      "S": "3006f9ea5a0904c9a53cd3e54cfd36c52d7e2132dedd37601e7e8926e80c7d42"
+    }
+  ],
+  "ExpectedSigHashes": [
+    "1b80a53fd3db13ebea43631419f38cc5e1c842fe6156793d9452c7bb6eeb5fbb",
+    "59aef86e6236d611e409ecbd1875d1f6bc33b03fac420501fce9d33b3509c87c"
+  ],
+  "ExpectedSweepTransaction": "01000000000102182d37bf3074663d099b1d7b2466311085f8ebfa27626a4c2802b43cf331ec220000000000ffffffff1caf1d0a247ec8a7094724669f86881ebbb13bdfdde0c5c77b02809ac43b87b30000000000ffffffff01f0951f00000000001600148db50eb52063ea9d98b3eac91489a90f738986f60247304402207a2c2165f22082cd35b78b606ccd9882c0d0b90eca4fc93cb4488875aadb1dd9022063f77b701286714c392627e9cc69238f8157e56dca4e24e397769c40ed1453d2012103989d253b17a6a0f41838b84ff0d20e8898f9d7b1a98f2564da4cc29dcf8581d903483045022100f7133a975f9728091fba874f3d6ff23d3c63c084cbb5e3e344c59a7910f9636802203006f9ea5a0904c9a53cd3e54cfd36c52d7e2132dedd37601e7e8926e80c7d42012103989d253b17a6a0f41838b84ff0d20e8898f9d7b1a98f2564da4cc29dcf8581d97e14934b98637ca318a4d6e7ca6ffd1690b8e77df6377520a9b38ea6435c8941d6eda6a46b68e3e2117196995bd154ab55196396b03d9bda7508f9f0c90d000395237576a9148db50eb52063ea9d98b3eac91489a90f738986f68763ac6776a91428e081f285138ccbe389c1eb8985716230129f89880460bcea61b175ac6800000000",
+  "ExpectedSweepTransactionHash": "d1ae6e4627aadcab4148469e88d43c65d15702b97890a9ae7dc03b06c8c677b7",
+  "ExpectedSweepTransactionWitnessHash": "eda66565064202484aaa10228ebc67a270ea51f6042b56ed8e028684a4271eb2"
+}
```

Comment:
Nitpicking: could you add an empty line?

Reply:
Please see: f5fc3e20